LibC/termcap.cpp

```diff
@@ -115,7 +115,7 @@ char* tgoto(const char* cap, int col, int row)
     (void)cap;
     (void)col;
     (void)row;
-    ASSERT_NOT_REACHED();
+    //ASSERT_NOT_REACHED();
```

Comment:
Just removing this seems wrong. It was before you were using vim's own termcap stuff, wasn't it?

Reply:
No, this prevents Bash and Less from crashing randomly.

Reply:
Then it sounds like it belongs in a separate PR, and it should also not just be commented out IMO, as it's definitely not correct to just stub functions without at least making it clear that they _are_ stubs, as then when trying to figure out why behaviour is incorrect, you have a much harder job as you don't know where to start looking :)

Either the function should be implemented, or at the least, it should be turned into a dbgprintf if the assert is to be removed -- but I'd say that leaving it as an assert is the safer option so it's clear that there is still work to do.